### DISCUSSION TITLE
Account hacked, and recovered, but still flagged.

### DISCUSSION THREAD

**Discussion by HonsalSub** [community/community#129977] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm reaching out due to persistent problems with my account that began after it was compromised. Despite updating all security measures and contacting support, the issues remain unresolved, affecting both my ability to use services and billing.

However that account that issue occurred is unable to write community post, so I am writing in backup account.

### Background:
- **Account Compromise:** My account was initially hacked, leading to unauthorized activities that were later detected, prompting an official security alert from GitHub.
- **Security Measures:** Following the alert, I promptly changed my password and updated all related security settings, including passkeys, 2FA, and SMS verification.
- **Access and Flagging:** After securing my account, I tried to access GitHub Copilot, which I had been subscribed to. However, I found the service inaccessible, my account was flagged, and I couldn't make any direct purchases from the plan comparison page due to this flag.

### Current Status:
- **Persistent Flagging:** Despite resolving the security issues, my account remains flagged, which blocks me from making purchases and accessing my subscriptions.
- **Billing Anomalies:** Ironically, while I cannot purchase anything or use my GitHub Copilot due to the flagged state, my subscription was still renewed and charged at date.
- **Lack of Support Response:** I have not received any updates or responses to my follow-up communications regarding support ticket (#2838675).

### Specific Issues:
- **Billing:** My account continues to be charged for services like the subscription renewal, despite being flagged and thus unable to access these services.
- **Access:** I am completely locked out of using essential tools like GitHub Copilot, which my work heavily relies on.

### Questions for the Community:
1. Has anyone else experienced prolonged flagging after resolving security issues from a hack? How did you resolve it?
2. What can be done to escalate the issue within GitHub when standard support responses fail?
3. Are there known issues or tips for dealing with account recovery and billing inconsistencies after security breaches?

**Comment by Nothing-Works** [comment]
I am having the same issue as you, I've been waiting for 2-3 months. I'm kind of giving up, as no one can help.

**Comment by Nevil844** [comment]
Hi there. I have a similar issue, any updates from the Github team?
